mindsdb_native/libs/data_sources/aws_athena_ds.py

```diff
@@ -1,54 +1,49 @@
 from pyathena import connect
 from pyathena.util import as_pandas
 
-from mindsdb_native.libs.data_types.data_source import DataSource
+from mindsdb_native.libs.data_types.data_source import SQLDataSource
 
 
-class AthenaDS(DataSource):
-    def __init__(self, *args, **kwargs):
-        self.is_sql = True
-        super(AthenaDS, self).__init__(*args, **kwargs)
-
-    def _setup(self, staging_dir, database=None, table=None, query=None,
-               access_key=None, secret_key=None, region_name=None):
+class AthenaDS(SQLDataSource):
```

Comment:
Looks like that API changes in data types also affect related code in mindsdb repo. see https://github.com/mindsdb/mindsdb/blob/01cef5e7a6ef41e4179bf2df7a0651b1dc43b343/mindsdb/interfaces/datastore/datastore.py#L112 as example. Are you going to perform them?

Reply:
right, I'll look into how these changes affect mindsdb before merging the PR